spec/bolt/project_migrator/modules_spec.rb

```diff
@@ -133,7 +133,7 @@ def make_moduledirs
       expect(migrate).to be(true)
       data = Bolt::Util.read_yaml_hash(project.project_file, 'project')
       expect(data['modules']).to match_array([
-                                               { 'name' => 'puppetlabs-yaml', 'version_requirement' => '0.1.0' }
+                                               { 'name' => 'puppetlabs/yaml', 'version_requirement' => '0.1.0' }
```

Comment:
It's probably useful to verify that both formats work, since they're both supported. It's probably best to have a test for that in on of the module_installer spec tests, but doing it here would also work.

Reply:
That's not what this test is checking. Since the module classes normalize the name, the project migrator will _always_ output names that match `owner/name`, it's not a matter of supporting `owner-name`, too.

The base `module` class already tests that `owner-name` is normalized, which seems sufficient for testing on our end, as `puppetfile-resolver` is what ultimately needs to support both formats and test that it works. The name is just normalized on our end so the Puppetfile output at least looks consistent. :P